test/integration/analytical_rosenbrock.cpp

```diff
@@ -91,7 +91,7 @@ TEST(AnalyticalExamples, BranchedSuperStiffButAnalytical)
 TEST(AnalyticalExamples, Robertson)
 {
   test_analytical_robertson(rosenbrock, 1e-1);
-  // test_analytical_robertson(backward_euler, 1e-1);
+  test_analytical_robertson(backward_euler, 1e-1);
```

Comment:
Interesting that backward euler solver could pass the test with the same tolerance as rosenbrock solver.

Reply:
I thought so, too. I think it's the way we are comparing the numbers, but anything I've tried always leaves us with the same tolerance. Maybe in resolving #578 we could come up with a better way to compare these numbers, or figure out some set of tolerances that make us match the analytical results directly

Reply:
Oh, I did not realize that this is for the Robertson test. We surely need to understand the #578 issue more first. Thanks for the clarificadtion.